types/compute-argmax/compute-argmax-tests.ts

```diff
@@ -0,0 +1,4 @@
+import compute = require('compute-argmax');
+
+const data = [3, 2, 5, 2, 5];
+const idx: number[] = compute.argmax(data);
```

Comment:
According to the usage and the underlying library [export](https://github.com/compute-io/argmax/blob/master/lib/index.js#L64) it looks like the function is exported directly. Changing the export in your `index.d.ts` to `export = ...` should fix it.

Reply:
Hi Pranav, thanks for the feedback

I am a little confused on the syntax, as

```
export = function argmax(values: number[]): number[];
```

tells me I am missing some curly braces {}

Reply:
oo I think I got it